demo/date-picker-basic-demos.html

```diff
@@ -6,141 +6,47 @@
-    <vaadin-demo-snippet id="date-picker-basic-demos-disabled-and-readonly">
+    <h3>Disabled and read-only</h3>
+    <vaadin-demo-snippet>
       <template preserve-content>
-        <vaadin-date-picker disabled label="Disabled input" value="1980-08-14"></vaadin-date-picker>
-        <vaadin-date-picker readonly label="Readonly input" value="1980-08-14"></vaadin-date-picker>
+        <vaadin-date-picker disabled label="Disabled" value="1990-01-28"></vaadin-date-picker>
+        <vaadin-date-picker readonly label="Read-only" value="1990-01-28"></vaadin-date-picker>
       </template>
     </vaadin-demo-snippet>
 
-
-    <h3>Initial position</h3>
-    <vaadin-demo-snippet id="date-picker-basic-demos-initial-position">
+    <h3>Clear button</h3>
+    <vaadin-demo-snippet>
       <template preserve-content>
-        <vaadin-date-picker label="Birthday" initial-position="1980-01-01">
-        </vaadin-date-picker>
+        <vaadin-date-picker clear-button-visible label="Label" value="1990-01-28"></vaadin-date-picker>
       </template>
     </vaadin-demo-snippet>
 
-
-    <h3>Date range</h3>
-    <p>Wrap two date pickers in <a href="https://vaadin.com/components/vaadin-custom-field"><code>&lt;vaadin-custom-field&gt;</code></a> to create a date range selector.</p>
-    <vaadin-demo-snippet id="date-picker-basic-demos-date-range" when-defined="vaadin-date-picker">
+    <h3>Value change event</h3>
+    <vaadin-demo-snippet id="value-change-event" when-defined="vaadin-date-picker">
       <template preserve-content>
-        <vaadin-custom-field label="Date range">
-          <vaadin-date-picker id="start"></vaadin-date-picker>
-          <vaadin-date-picker id="end"></vaadin-date-picker>
-        </vaadin-custom-field>
-        <p>Selected range: <span></span></p>
+        <vaadin-date-picker label="Label"></vaadin-date-picker>
+        <div>Selected value: <span id="output"></span></div>
         <script>
-          window.addDemoReadyListener('#date-picker-basic-demos-date-range', function(document) {
-            const start = document.querySelector('#start');
-            const end = document.querySelector('#end');
-            const span = document.querySelector('span');
-            const dateRange = document.querySelector('vaadin-custom-field');
-
-            start.addEventListener('change', function() {
-              end.min = start.value;
-
-              // Open the second date picker when the user has selected a value
-              if (start.value) {
-                end.open();
-              }
-            });
-
-            end.addEventListener('change', function() {
-              start.max = end.value;
-            });
-
-            dateRange.addEventListener('change', function() {
-              span.textContent = dateRange.value.trim().replace(/\t/, ' - ');
+          window.addDemoReadyListener('#value-change-event', function(document) {
+            const comboBox = document.querySelector('vaadin-date-picker');
```

Comment:
`comboBox` variable name could be changed.

Reply:
Done.